xml/zypper_upgrade.xml

```diff
@@ -67,119 +67,149 @@
-       clean upgrade path for all your packages.
+       Repeat this for all repositories you no longer need.
       </para>
      </step>
      <step>
       <para>
-       Disable third party repositories or other &obs; repositories, because
-       <command>zypper dup</command> is guaranteed to work with the default
-       repositories only (replace <replaceable>REPO-ALIAS</replaceable> with
-       the name of the repository you want to disable):
-      </para>
-<screen>&prompt.sudo;zypper mr -d <replaceable>REPO-ALIAS</replaceable></screen>
-      <para>
-       Alternatively, you can lower the priority of these repositories.
+       In case you still need a repository, continue as follows.
       </para>
-      <note>
-       <title>Handling of unresolved dependencies</title>
-       <para>
-        <command>zypper dup</command> will remove all packages having
-        unresolved dependencies, but it keeps packages of disabled
-        repositories as long as their dependencies are satisfied.
-       </para>
-      </note>
       <para>
-       <command>zypper dup</command> ensures that all installed packages come
-       from one of the available repositories. It does not consider the version
-       or architecture, but prevents changing the vendor of the installed
-       packages by default, using the <option>--no-allow-vendor-change</option>
-       option. Packages that are no longer available in the repositories are
-       considered orphaned. Such packages get uninstalled if their dependencies
-       cannot be satisfied. If they can be satisfied, such packages stay
-       installed.
+       For 3rd party repositories that do not include a version number, check
+       with the provider of the repository wether &productname;
+       &productnumber; is supported. If so, keep the repository. If not,
+       remove it as described above.
       </para>
-     </step>
-     <step>
       <para>
-       Once done, check your repository configuration with:
+       All remaining repositories you want to keep need to be updated to the
+       new version. For this you need to remove the old version and add the
+       new one. This procedure is done in the following with an example
+       repository that has the following data:
       </para>
-<screen>&prompt.user;zypper lr -d</screen>
+      <simplelist>
+       <member>Name: <literal>M17N:fonts</literal></member>
+       <member>URI: <literal>https://download.opensuse.org/repositories/M17N:/fonts/openSUSE_Leap_15.2/</literal></member>
+      </simplelist>
+      <substeps>
+       <step>
+        <para>
+         Remove the old repository:
+        </para>
+        <screen>&prompt.sudo;zypper rr "M17N:fonts"</screen>
+       </step>
+       <step>
+        <para>
+         Add the new repository:
+        </para>
+        <screen>&prompt.sudo;zypper ar -f \
+        https://download.opensuse.org/repositories/M17N:/fonts/openSUSE_Leap_15.3/ \
+        "M17N:fonts"</screen>
```

Comment:
This will create unwanted indentation in the output. Fix:

```suggestion
  https://download.opensuse.org/repositories/M17N:/fonts/openSUSE_Leap_15.3/ \
  "M17N:fonts"</screen>
```